docs/depfile.md

```diff
@@ -143,8 +149,9 @@ section.
 Example:
 
 ```yaml
-imports:
-  - some/depfile.yaml
+parameters:
+  imports:
```

Comment:
Reminder to self: This needs to be fixed (both here and in the Config Loader).

Imports should be a top level section just like parameters. It's only a small adjustment in the Loader, but we need to make it clear in the Upgrade guide, that this is the only thing that is not moved to parameters.